app/controllers/application_controller.rb

```diff
@@ -11,38 +11,15 @@ class ApplicationController < ActionController::Base
   private
   def load_context
     context_id = params[:context_id]
-    country_name = params[:country]
-    country_id = params[:country_id]
-    commodity_name = params[:commodity]
-    commodity_id = params[:commodity_id]
 
-    if (not context_id.nil?)
+    if context_id.present?
       @context = Context.find(context_id)
 
       raise ActionController::ParameterMissing, 'The provided context could not be found' if @context.nil?
```

Comment:
actually this line will never execute - if context is not found, find will throw an exception